unified-codes/.babelrc

```diff
@@ -1,4 +1,12 @@
 {
   "presets": ["@babel/preset-env"],
-  "plugins": [["@babel/transform-runtime"]]
+  "plugins": [
+    ["@babel/transform-runtime"],
+    [
+      "@babel/plugin-proposal-class-properties",
```

Comment:
I just spent 20 mins getting this working! Should have checked...

![kiddingme](https://user-images.githubusercontent.com/43223637/90368112-b2b70100-e0bd-11ea-84aa-45bbf3f905e0.gif)